engine/CMakeLists.txt

```diff
@@ -531,9 +531,9 @@ IF (NOT BUILD_OPT)
     SET(BUILD_OPT -O2 CACHE STRING "-O0, -O1, -O2, -O3, -Os, -Ofast" FORCE )
 ENDIF (NOT BUILD_OPT)
 
-IF (CMAKE_COMPILER_IS_GNUCXX)
-    ADD_DEFINITIONS(-std=gnu++03)
-ENDIF ()
+# IF (CMAKE_COMPILER_IS_GNUCXX)
+#     ADD_DEFINITIONS(-std=gnu++03)
+# ENDIF ()
```

Comment:
I believe we're setting Cxx11 elsewhere so this can go away entirely.

Reply:
Yes. This can probably get done in a future PR.